include/seqan/rna_io/bpseq_read_write.h

```diff
@@ -0,0 +1,235 @@
+// Function readRecord()                                          [BpseqRecord]
+// ----------------------------------------------------------------------------
+
+template <typename TForwardIter>
+inline void
+readRecord(RnaRecord & record, TForwardIter & iter, Bpseq const & /*tag*/)
+{
+    typedef OrFunctor<IsSpace, AssertFunctor<NotFunctor<IsNewline>, ParseError, Bpseq> > NextEntry;
+    std::string buffer;
+    clear(record);
+
+    skipUntil(iter, NotFunctor<IsWhitespace>());
+    while (!atEnd(iter) && value(iter) == '#')
+    {                                       // All the information stored in the # lines are saved in a single line
+        skipOne(iter);
+        skipUntil(iter, NotFunctor<IsWhitespace>());
+        readLine(buffer, iter);
+        if (empty(record.name))
+        {
+            record.name = buffer;
+        }
+        else
+        {
+            appendValue(record.comment, ' ');
+            append(record.comment, buffer);
+        }
+        clear(buffer);
+        skipUntil(iter, NotFunctor<IsWhitespace>());
+    }
+
+    RnaStructureGraph graph;
+    unsigned currPos{0};
+    while (!atEnd(iter) && value(iter) != '#')
+    {
+        // read index position
+        if (currPos == 0)
+        {
+            readUntil(buffer, iter, NextEntry());
+            if (empty(buffer))
+                SEQAN_THROW(EmptyFieldError("BEGPOS"));
+            if (!lexicalCast(record.offset, buffer))
+                SEQAN_THROW(BadLexicalCast(record.offset, buffer));
+            currPos = record.offset;
+            clear(buffer);
+        }
+        else
+        {
+            skipUntil(iter, NextEntry());
+            ++currPos;
+        }
+
+        // read nucleotide
+        skipUntil(iter, NotFunctor<IsBlank>());
+        readUntil(buffer, iter, NextEntry());
+        appendValue(record.sequence, buffer[0]);
+        addVertex(graph.inter);                 // add base to graph
+        if (empty(buffer))
+            SEQAN_THROW(EmptyFieldError("SEQUENCE"));
+        clear(buffer);
+
+        // read paired index
+        skipUntil(iter, NotFunctor<IsBlank>());
+        readUntil(buffer, iter, IsWhitespace());
+        if (empty(buffer))
+            SEQAN_THROW(EmptyFieldError("PAIR"));
+        unsigned pairPos;
+        if (!lexicalCast(pairPos, buffer))
+            SEQAN_THROW(BadLexicalCast(pairPos, buffer));
+        if (pairPos != 0 && currPos > pairPos)    // add edge if base is connected
+            addEdge(graph.inter, pairPos - record.offset, currPos - record.offset, 1.);
+        clear(buffer);
+
+        skipLine(iter);
+        skipUntil(iter, NotFunctor<IsWhitespace>());
+    }
+    append(record.fixedGraphs, graph);
+    record.seqLen = currPos;  //set amount of records
+}
+
+template <typename TForwardIter>
+inline void
+readRecord(RnaRecord & record, SEQAN_UNUSED RnaIOContext &, TForwardIter & iter, Bpseq const & /*tag*/)
```

Comment:
I thought the macro was meant to indicate function arguments which are unused on purpose... Now I delete the macro everywhere I used it like this.